### DISCUSSION TITLE
ECDSA example runs slower than expected?

### DISCUSSION THREAD

**Discussion by matthiasgeihs** [risc0/risc0#1389] [Q&A] [answered]
I've been running the [ECDSA example](https://github.com/risc0/risc0/tree/main/examples/ecdsa) on an M1 Pro (10/16 cores) with Metal enabled (`cargo run --release --features=risc0-zkvm/metal`) at latest commit (83cf6867025846cd8ec91d9776ef8fb6c74b3b88).

The README suggests that this should take about `17s`. However, I'm measuring a prover running time of about `52s`. Where is that difference coming from?

**Comment by pdg744** [comment, marked as answer]
This performance hit was caused by the v0.20 release and addressed in the v0.20.1 release.
